razzle.config.js

```diff
@@ -186,12 +191,23 @@ const plugins = addonExtenders.reduce(
 
 module.exports = {
   plugins,
-  modify: (config, { target, dev }, webpack) => {
-    const defaultConfig = defaultModify(config, { target, dev }, webpack);
+  modifyWebpackConfig: ({
```

Comment:
Maybe at some point, later, we could also support the additional plugin entry points for Razzle, coming from razzle.extend.js files. My proposal would be: keep the `modify()` as modifyWebpackConfig, but check if the extend file implements any of the other methods.

Reply:
Unfortunately, if we do that, Razzle complains and throw a warning, and I guess then we won't be Razzle 4 compat. We can still support backwards compat by looking into the modify property, then treat it as the modifyWebpackConfig (same as Razzle 3 is doing now).